src/sentry/tasks/sentry_apps.py

```diff
@@ -189,12 +189,11 @@ def _process_resource_change(action, sender, instance_id, retryer=None, *args, *
         data = {}
         if isinstance(instance, Event):
             data[name] = _webhook_event_data(instance, instance.group_id, instance.project_id)
-            send_webhooks(installation, event, data=data)
         else:
             data[name] = serialize(instance)
-            send_webhooks(installation, event, data=data)
 
-        metrics.incr("resource_change.processed", sample_rate=1.0, tags={"change_event": event})
+        # Trigger a new task for each webhook
+        send_resource_change_webhook.delay(installation=installation, event=event, data=data)
```

Comment:
@NisanthanNanthakumar if this change doesn't break a test, then we need to add a test for this speec

Reply:
@scefali yeah it should not break any tests, since the functionality is the same. But I can definitely add a test to make sure this task gets created.

Reply:
@NisanthanNanthakumar well, we should add a test for the case where one installation fails which was the original bug.